ckan-deployment/helm-templates/values.yaml.prod.template

```diff
@@ -29,38 +26,34 @@ ckan:
       CKAN__HARVEST__MQ__TYPE: redis
       CKAN__LOCALE_DEFAULT: en
       CKAN__RESOURCE_PROXY__MAX_FILE_SIZE: "100000000"
-      CKAN__SITE_TITLE: x demo
+      CKAN__SITE_TITLE: WRI Open Data Platform
       CKAN__STORAGE_PATH: /tmp
       CKAN__VIEWS__DEFAULT_VIEWS: recline_view text_view image_view pdf_view geojson_view
-      CKAN__PLUGINS: image_view text_view webpage_view resource_proxy datatables_view datastore datapusher activity envvars
+      CKAN__PLUGINS: image_view text_view webpage_view resource_proxy datatables_view datastore datapusher activity s3filestore envvars
       CKAN__AUTH__CREATE_UNOWNED_DATASET: "True"
-      CKAN__AUTH__ALLOW_DATASET_COLLABORATORS: "True"
-      CKAN__AUTH__ALLOW_ADMIN_COLLABORATORS: "True"
-      CKAN__AUTH__ALLOW_COLLABORATORS_TO_CHANGE_OWNER_ORG: "True"
+      CKAN__AUTH__ALLOW_DATASET_COLLABORATORS: True
```

Comment:
I think these quotes need to be added back in (you fixed this previously).

Reply:
Good catch.
Ill add the quotes

Reply:
Resolved